warp-core/src/test/scala/com/workday/warp/persistence/ConcurrentWriteTest.scala

```diff
@@ -50,7 +50,7 @@ class ConcurrentWriteTest extends WarpJUnitSpec with CorePersistenceAware {
 
   @UnitTest
   def concurrentTestExecution(): Unit = {
-    using threads 8 trials 32 invoke {
+    using threads 8 trials 16 invoke {
```

Comment:
Why did you scale it down?

Reply:
this test fails sporadically in github actions CI, i think just because of limited resources in the build environment